homeassistant/components/eq3btsmart/icons.json

```
@@ -0,0 +1,36 @@
+{
+  "entity": {
+    "binary_sensor": {
+      "dst": {
+        "default": "mdi:sun-clock",
+        "state": {
+          "on": "mdi:sun-clock",
+          "off": "mdi:sun-clock-outline"
+        }
+      }
```

```suggestion
    "binary_sensor": {
      "dst": {
        "default": "mdi:sun-clock",
        "state": {
          "off": "mdi:sun-clock-outline"
        }
      }
```
It will fall back to the default

Also, you can remove the icon attribute from the binary sensor entity description